### PR TITLE
Improve bug report template description

### PR DESCRIPTION
**Pull request by ssbarnea**
Attempt to lower the number of false-positive bug reports, about
problems that happen only with older versions.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: 🐛 Bug report
-about: Create a report to help us improve
+about: Create a bug report, only if you can reproduce on master branch.
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Create a bug report (only if it's reproducible on the master branch)
```

Comment (ssbarnea) [reply .github/ISSUE_TEMPLATE/bug_report.md:3]: @gundalow What do you think? Which one fits better with Cambridge English? ;)

Comment (ssbarnea) [reply .github/ISSUE_TEMPLATE/bug_report.md:3]: @awcrosby What do you think? Comma or parentheses for the descriptions used on https://github.com/ansible/ansible-lint/issues/new/choose ?

[the remark this answers is not in the document]
Comment (webknjaz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This introduces a bit of inconsistency with a trailing dot. Other options don't have it.

**Pull request merged**